src/Products/Viewer/Controllers/ViewerApiController.cs

```diff
@@ -197,53 +206,57 @@ public HttpResponseMessage LoadDocumentPage(PostedDataEntity postedData)
         /// </summary>
         /// <param name="postedData">Document page number to rotate and rotation angle</param>
         /// <returns>Rotated document page object</returns>
-        [HttpPost]
-        [Route("rotateDocumentPages")]
-        public HttpResponseMessage RotateDocumentPages(PostedDataEntity postedData)
-        {
-            try
-            {
-                // get/set parameters
-                string documentGuid = postedData.guid;
-                int angle = postedData.angle;
-                List<int> pages = postedData.pages;
-                string password = postedData.password;
-                DocumentInfoOptions documentInfoOptions = new DocumentInfoOptions(documentGuid);
-                // set password for protected document                
-                documentInfoOptions.Password = password;
-                // a list of the rotated pages info
-                List<RotatedPageEntity> rotatedPages = new List<RotatedPageEntity>();
-                // rotate pages
-                for (int i = 0; i < pages.Count; i++)
-                {
-                    // prepare rotated page info object
-                    RotatedPageEntity rotatedPage = new RotatedPageEntity();
-                    int pageNumber = pages[i];
-                    RotatePageOptions rotateOptions = new RotatePageOptions(pageNumber, angle);
-                    // perform page rotation
-                    string resultAngle = "0";
-                    // set password for protected document
-                    if (!string.IsNullOrEmpty(password))
-                    {
-                        rotateOptions.Password = password;
-                    }
-                    this.GetHandler().RotatePage(documentGuid, rotateOptions);
-                    resultAngle = this.GetHandler().GetDocumentInfo(documentGuid, documentInfoOptions).Pages[pageNumber - 1].Angle.ToString();
-                    // add rotated page number
-                    rotatedPage.SetPageNumber(pageNumber);
-                    // add rotated page angle
-                    rotatedPage.SetAngle(resultAngle);
-                    // add rotated page object into resulting list                   
-                    rotatedPages.Add(rotatedPage);
-                }
-                return Request.CreateResponse(HttpStatusCode.OK, rotatedPages);
-            }
-            catch (System.Exception ex)
-            {
-                // set exception message
-                return Request.CreateResponse(HttpStatusCode.InternalServerError, new Resources().GenerateException(ex));
-            }
-        }
+        //[HttpPost]
+        //[Route("rotateDocumentPages")]
+        //public HttpResponseMessage RotateDocumentPages(PostedDataEntity postedData)
+        //{
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/gh/groupdocs-viewer/GroupDocs.Viewer-for-.NET-MVC/pullRequest?prid=5242614)